From: Michael Hanke <mih@debian.org>
Subject: Rename duplicate symbol

libutils code has two symbols of identical name. Disable a randomly chosen one
(need to clarify which one is the intended one). In the static libs ones of them
gets swallowed.
--- a/src/condor_utils/signames.cpp
+++ b/src/condor_utils/signames.cpp
@@ -19,7 +19,10 @@
 
 #include "condor_common.h"
 
-const char	*SigNames[] = {
+/* there is another 'SigNames' src/condor_utils/event_handler.unix.cpp
+ * that has been chosen by fair flip of a coin
+ */
+const char	*SigNamesDISABLED[] = {
 "(0)",
 "SIGHUP",		/* 1 */
 "SIGINT",		/* 2 */
